adminer/lang/de.inc.php

```diff
@@ -33,7 +33,7 @@
 	'SQL command' => 'SQL-Kommando',
 	'Logout' => 'Abmelden',
 	'database' => 'Datenbank',
-	'Use' => 'Benutzung',
```

Comment:
It depends if it is used by Adminer code/template as noun or verb in German. 

'Use' as a noun: Benutzung
'use' a a verb: benutzen  (if used as a button title you may uppercase it, but that is a matter of taste)

Reply:
It is used solely as verb. So "Benutzen" is correct.